docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -227,7 +227,7 @@ children:
       output: 'web, pdf'
       children:
         - title: 'Apply JWT Authentication in Todo Example'
-          url: Authentication-tutorial.html
+          url: Authentication-Tutorial.html
```

Comment:
I think the convention is to only have the 1st letter to be upper case.

Reply:
ok. let me change the file name instead of the reference.